letta/agent.py

```diff
@@ -551,18 +551,15 @@ def _handle_ai_response(
             )  # extend conversation with assistant's reply
             printd(f"Function call message: {messages[-1]}")
 
-            # The content if then internal monologue, not chat
-            self.interface.internal_monologue(response_message.content, msg_obj=messages[-1])
```

Comment:
Can you replace this with what you have below:

```python
            if response_message.content:
                # The content if then internal monologue, not chat
                self.interface.internal_monologue(response_message.content, msg_obj=messages[-1])
```

Reply:
Ack